content/strategy-goals/strategy/code-graph/search/core.md

```diff
@@ -41,6 +46,8 @@ We also agreed that growing the global index from 2.1M to 5.5M wouldn't have mad
 
 In addition, while our work on scaling had produced trickle-down benefits on performance and efficiency for our Enterprise deployments, the work we would have produced to scale from 2.1M to 5.5M would have been mostly Cloud-specific.
 
+We've chosen to make progress towards [indexing the entire OSS universe](https://about.sourcegraph.com/blog/why-index-the-oss-universe/) by indexing repositories from different code hosts. By focusing on code host coverage, we strive to make Sourcegraph.com a truly universal code search engine: Sourcegraph will not be limited to searching over just the most popular code hosts thus increasing the discoverability and visibility of OSS code regardless of the code host. We see as a better increment towards making sourcegraph.com a compelling OSS code search engine than our previous goal of scaling the global index to 5.5M repositories.
```

Comment:
This part seems to describe what we are doing and probably doesn't belong in the what we aren't doing section.